app/src/test/java/ch/hikemate/app/navigation/NavigationActionsTest.kt

```diff
@@ -0,0 +1,114 @@
+package ch.hikemate.app.navigation
+
+import android.content.Context
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.AccountBox
+import androidx.navigation.NavDestination
+import androidx.navigation.NavGraph
+import androidx.navigation.NavHostController
+import androidx.navigation.NavOptionsBuilder
+import androidx.navigation.navOptions
+import androidx.navigation.testing.TestNavHostController
+import ch.hikemate.app.ui.navigation.NavigationActions
+import ch.hikemate.app.ui.navigation.Route
+import ch.hikemate.app.ui.navigation.Screen
+import ch.hikemate.app.ui.navigation.TopLevelDestination
+import ch.hikemate.app.ui.navigation.TopLevelDestinations
+import org.hamcrest.CoreMatchers.`is`
+import org.hamcrest.MatcherAssert.assertThat
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mockito.mock
+import org.mockito.Mockito.spy
+import org.mockito.Mockito.verify
+import org.mockito.Mockito.`when`
+import org.mockito.kotlin.any
+import org.mockito.kotlin.argumentCaptor
+import org.mockito.kotlin.eq
+
+class NavigationActionsTest {
+
+  private lateinit var navigationDestination: NavDestination
+  private lateinit var navHostController: NavHostController
+  private lateinit var navigationActions: NavigationActions
+
+  private lateinit var navGraph: NavGraph
+  private lateinit var navController: TestNavHostController
+  private lateinit var context: Context
+  private var navOptionsCaptor = argumentCaptor<NavOptionsBuilder.() -> Unit>()
+
+  @Before
+  fun setUp() {
+    context = mock(Context::class.java)
+    navigationDestination = mock(NavDestination::class.java)
+    navHostController = mock(NavHostController::class.java)
+    navigationActions = NavigationActions(navHostController)
+    navGraph = mock(NavGraph::class.java)
+    `when`(navHostController.graph).thenReturn(navGraph)
+    navController = spy(TestNavHostController(context))
+  }
+
+  @Test
+  fun navigateToCallsController() {
+    navigationActions.navigateTo(TopLevelDestinations.OVERVIEW)
+    verify(navHostController)
+        .navigate(eq(Route.OVERVIEW), org.mockito.kotlin.any<NavOptionsBuilder.() -> Unit>())
```

Comment:
NitPicking: AndroidStudio tells me that you don't need to specify `org.mockito.kotlin`
```suggestion
        .navigate(eq(Route.OVERVIEW),any<NavOptionsBuilder.() -> Unit>())
```